bitnami/cilium/values.yaml

```diff
@@ -0,0 +1,1515 @@
+  ## @param agent.containerPorts.pprof Cilium Agent pprof container port
+  ## @param agent.containerPorts.gops Cilium Agent gops container port
+  ## @param agent.containerPorts.hubblePeer Cilium Agent Hubble peer service container port
+  ## @param agent.containerPorts.metrics Cilium Agent metrics container port
+  ## @param agent.containerPorts.hubbleMetrics Cilium Agent Hubble metrics container port
+  ##
+  containerPorts:
+    health: 9879
+    gops: 9890
+    pprof: 6060
+    hubblePeer: 4244
+    metrics: 9962
+    hubbleMetrics: 9965
+  ## @param agent.extraContainerPorts Optionally specify extra list of additional ports for Cilium Agent containers
+  ## e.g:
+  ## extraContainerPorts:
+  ##   - name: myservice
+  ##     containerPort: 9090
+  ##
+  extraContainerPorts: []
+  ## Configure extra options for Cilium Agent containers' liveness and readiness probes
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/#configure-probes
+  ## @param agent.livenessProbe.enabled Enable livenessProbe on Cilium Agent containers
+  ## @param agent.livenessProbe.initialDelaySeconds Initial delay seconds for livenessProbe
+  ## @param agent.livenessProbe.periodSeconds Period seconds for livenessProbe
+  ## @param agent.livenessProbe.timeoutSeconds Timeout seconds for livenessProbe
+  ## @param agent.livenessProbe.failureThreshold Failure threshold for livenessProbe
+  ## @param agent.livenessProbe.successThreshold Success threshold for livenessProbe
+  ##
+  livenessProbe:
+    enabled: true
+    initialDelaySeconds: 10
+    periodSeconds: 30
+    timeoutSeconds: 5
+    failureThreshold: 10
+    successThreshold: 1
+  ## @param agent.readinessProbe.enabled Enable readinessProbe on Cilium Agent containers
+  ## @param agent.readinessProbe.initialDelaySeconds Initial delay seconds for readinessProbe
+  ## @param agent.readinessProbe.periodSeconds Period seconds for readinessProbe
+  ## @param agent.readinessProbe.timeoutSeconds Timeout seconds for readinessProbe
+  ## @param agent.readinessProbe.failureThreshold Failure threshold for readinessProbe
+  ## @param agent.readinessProbe.successThreshold Success threshold for readinessProbe
+  ##
+  readinessProbe:
+    enabled: true
+    initialDelaySeconds: 10
+    periodSeconds: 30
+    timeoutSeconds: 5
+    failureThreshold: 3
+    successThreshold: 1
+  ## @param agent.startupProbe.enabled Enable startupProbe on Cilium Agent containers
+  ## @param agent.startupProbe.initialDelaySeconds Initial delay seconds for startupProbe
+  ## @param agent.startupProbe.periodSeconds Period seconds for startupProbe
+  ## @param agent.startupProbe.timeoutSeconds Timeout seconds for startupProbe
+  ## @param agent.startupProbe.failureThreshold Failure threshold for startupProbe
+  ## @param agent.startupProbe.successThreshold Success threshold for startupProbe
+  ##
+  startupProbe:
+    enabled: true
```

Comment:
We don't enable startupProbes by default. As all the logic is in the init containers, I don't see much value.

Reply:
done at https://github.com/bitnami/charts/pull/26374/commits/129afd439cbffb07203c84e7db2880f82caa370f